packages/tailwindcss/src/index.test.ts

```diff
@@ -5287,6 +5286,10 @@ describe('`color-mix(…)` polyfill', () => {
         }
       }
 
+      .shadow-xl {
+        box-shadow: var(--tw-inset-shadow), var(--tw-inset-ring-shadow), var(--tw-ring-offset-shadow), var(--tw-ring-shadow), var(--tw-shadow);
+      }
```

Comment:
Interesting, what happened here 🤔

Reply:
Yeah that's fair, just an interesting change. I would be smaller again if we didn't have the `@supports` but this is fine (and correct) so let's merge it!